python/pyproject.toml

```diff
@@ -38,7 +38,7 @@ runtime_common = [
     "pyzmq>=25.1.2",
     "soundfile==0.13.1",
     "torchao>=0.7.0",
-    "transformers==4.51.1",
+    "transformers @ git+https://github.com/huggingface/transformers.git@v4.51.3-Qwen2.5-Omni-preview",
```

Comment:
This is not allowed. You can print a warning for people who want to use qwen omni